q2_quality_filter/plugin_setup.py

```diff
@@ -78,7 +79,8 @@
 
 plugin.methods.register_function(
     function=q2_quality_filter.q_score,
-    inputs={'demux': SampleData[SequencesWithQuality]},
+    inputs={'demux': (SampleData[SequencesWithQuality] |
```

Comment:
For the sake of consistency, can we follow what is [already in place in q2-dada2](https://github.com/qiime2/q2-dada2/blob/master/q2_dada2/plugin_setup.py#L39-L40)?